types/externs.d.ts

```diff
@@ -8,7 +8,7 @@ import _Crdp from 'devtools-protocol/types/protocol';
 import _CrdpMappings from 'devtools-protocol/types/protocol-mapping'
 
 // Import for side effects improving types of querySelector/querySelectorAll.
```

Comment:
I'm guessing this comment isn't true anymore based on your explanation and the below typedefs?

```suggestion
```